### ISSUE TITLE
Added two friends, didn't save, match thinks it is already full

### ISSUE THREAD

**Issue by antn** [cheshire137/competiwatch#85]
I added two teammates on this match:
https://www.competiwatch.com/matches/9/Am%C3%A9lie-11255/8088

Then, for my next one, I think they were displayed as checkboxes. So I added them, but then I got an error and couldn't save the match:
https://www.competiwatch.com/matches/9/Am%C3%A9lie-11255/8090

And trying to add them again gives me this error:

![image](https://user-images.githubusercontent.com/858504/37573516-f8842776-2ad5-11e8-8a44-74a8b20f4781.png)

Not sure what I did wrong 😅

**Comment by cheshire137** [comment]
Hmm, were you checking an additional checkbox or adding another person in the text field? Your screenshot only has five people chosen, three via checkbox and two in the text field, which should be valid.

**Comment by antn** [comment]
Nope, that's what's weird. What originally happened is that in match 8088 I added those two, and it saved fine, no error displayed.

Then I went to log the next match and the users aren't displayed as checkbox options. So I thought that I forgot to add them or something, and entered their name. Then I got this error.

Looking back at 8088, those two users are no longer displayed at all. Both of these matches give me this error when trying to enter the names, since the checkboxes for them aren't displayed. Are these users somehow associated with these matches, but aren't being displayed somehow?

I can reproduce this 100% of the time when trying to edit those matches to add sazuki and choo.
